editor/plugins/script_editor_plugin.cpp

```diff
@@ -3960,7 +3955,7 @@ void ScriptEditorPlugin::edit(Object *p_object) {
 		Script *p_script = Object::cast_to<Script>(p_object);
 		String res_path = p_script->get_path().get_slice("::", 0);
 
-		if (p_script->is_built_in()) {
+		if (p_script->is_built_in() && res_path != "") {
```

Comment:
```suggestion
		if (p_script->is_built_in() && !res_path.is_empty()) {
```
IIRC this was established as preferred style.